ocp_resources/pipelineruns.py

```diff
@@ -0,0 +1,57 @@
+# API reference: https://tekton.dev/docs/pipelines/pipelineruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class PipelineRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        pipelineref=None,
+        params=None,
+        service_account_name=None,
+        client=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            name (str): Name of the pipelinerun
+            namespace (str): Namespace of the pipelinerun
+            pipelineref (str): Mandatory: Base pipeline to run pipelineruns
+            client: (DynamicClient): DynamicClient to use.
+            params (dict): Optional params to add during triggering a run.
+            params can be set/changed based on pipelineref.
+            example : params={"param_name1":"param_value1", "param_name2":"param_value2"}
+            Refer : https://tekton.dev/docs/pipelines/pipelineruns/#specifying-parameters
```

Comment:
There's already a reference to the API, here I'd expect something like params={"pl-param-x": "100"} to match the above spec.

Reply:
ack